NEWS

```diff
@@ -1,3 +1,9 @@
+Version 1.3:
+	Activate -n switch in gzip program
+
+	Add file_name parameter to libdeflate_gzip_compress function. This
```

Comment:
Why isn't the decompression case implemented?  If this feature is offered at all, I think people would expect that it be supported on both the compression and decompression side.